src/pipeline_v3/tests/integration/test_e2e_integration.py

```
@@ -78,6 +84,10 @@
 
             try:
                 # Test document addition
+                print(f"About to process document: {doc_path}")
+                print(f"Pipeline config - OpenAI API key present: {bool(pipeline.config.openai_api_key)}")
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)